Server/src/diagramcoordinator.erl

```diff
@@ -85,9 +86,16 @@ spawn_node(Class_name, Did, Coordinator) ->
 
 %sends a message to the given node
 send_message(Receiver, From, To, Message, To_pid, Message_number, Coordinator, Did) ->
-  Receiver ! {send_message, From, To, Message, To_pid, Message_number},
-  receive
-    {send_reply, From, To, Message, To_pid, Message_number} -> 
-		%Sends info to the Coordinator, that the node sucessfully sent a message. To be printed in the executionlog	
-		Coordinator ! {Did, print_information, ["Node " ++ atom_to_list(From) ++ " sent a message to " ++ atom_to_list(To)]} 
-		end.
+
+	% Checks if there is a valid pid provided.o
```

Comment:
.o? Other than that, code looks good from what I can see. Haven't had the time to test it yet though, will do that as soon as possible